src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return {
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
+    ownerCount: data.OwnerCount,
+    previousTransactionID: data.PreviousTxnID,
+    previousTransactionLedgerVersion: data.PreviousTxnLgrSeq
```

Comment:
I thought I saw you supporting AccountTxnID for Flags elsewhere? It's a `thing` in RippleAPI ?